docs/DeveloperGuide.md

```diff
@@ -209,6 +209,38 @@ The following sequence diagram shows how the batch delete operation works:
 * Besides, if users leave an insurance company, they may like to delete people purchase policy from that company. 
 * Therefore, `Model#batchDeleteWithPredicate(Predicate<Person> predicate)` is introduced to allow batch delete by month or company.
 
+<br>
+
+### Edit Feature
+
+The `edit` command allows users to edit the details of a person in the address book.
+
+#### Implementation:
+
+The edit mechanism is facilitated by `EditCommand`. It extends `Command` with an `EditPersonDescriptor` to store the details of the person to be edited. 
+
+Additionally, it implements the following operations:
+* `EditCommand#createEditedPerson(Person personToEdit, EditPersonDescriptor editPersonDescriptor)` — Creates and returns a `Person` with the details of `personToEdit` edited with `editPersonDescriptor`.
+* `EditCommand#execute(Model model)` — Edits the details of the `Person` in the address book.
+
+The changes are finally made to the address book by calling `Model#setPerson(Person target, Person editedPerson)`.
+
+Given below is the sequence diagram for the `edit` command:
+
+![EditFeatureSequenceDiagram](images/EditFeatureSequenceDiagram1.png)
+
+:information_source: **Note:** The lifeline for `EditCommandParser` and `EditCommand` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```

Comment:
Good to have this note! Shows the limitation of PlantUML so that the other developers that reads it won't be confused.